Cars/Cars/Controllers/AdminPanelController.cs

```diff
@@ -59,11 +55,10 @@ public ViewResult Create()
         [HttpPost]
         public ActionResult Delete(int CarId)
         {
-            Car deletedCar = _allCars.DeleteCar(CarId);
+            Car deletedCar = _Car.DeleteCar(CarId);
```

Comment:
_Car.DeleteCar(CarId) should return bool and if should looks like 

**Bool isDeleted = _Car.DeleteCar(CarId)
if (isDeleted) { ... }**
     or similar